app/scripts/views/NodeViews/Num.js

```diff
@@ -69,21 +69,23 @@ define(['backbone', 'underscore', 'jquery', 'BaseNodeView', 'jqueryuislider'], f
 
       // adjust settings dropdown so that it stays open while editing
       // doesn't select the node when you're editing
-      $('.dropdown.keep-open').on({
+      this.$el.find('.dropdown.keep-open').on({
         "shown.bs.dropdown": function() {
           this.selectable = false;
           this.model.set('selected', false);
-          $('.dropdown.keep-open').data('closable', false);
+          this.$el.find('.dropdown.keep-open').data('closable', false);
```

Comment:
`$('.dropdown.keep-open')` will give elements from all nodes when we need only one from current node